doc/source/whatsnew/v1.0.0.rst

```diff
@@ -39,6 +39,28 @@ Backwards incompatible API changes
 - :class:`pandas.core.groupby.GroupBy.transform` now raises on invalid operation names (:issue:`27489`).
 -
 
+.. _whatsnew_0250.api_breaking.series_string_conversion:
+
+String conversion of Series with nan
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:meth:Series.astype(str) previously would coerce a np.nan to the string nan. Now pandas will preserve the missing value indicator (:issue:`25353`)
```

Comment:
IIUC, @jschendel and I both prefer a deprecation cycle. It looks like you're making a breaking change. Is that correct?